test/versioned/mongodb-esm/db.tap.mjs

```diff
@@ -287,11 +294,11 @@ tap.test('Db tests', (t) => {
 
   t.test('renameCollection', (t) => {
     dbTest({ t, agent, mongodb }, function collectionTest(t, db, verify) {
```

Comment:
This is the only functional change, right? Everything else is just cosmetic to make it more tidy?

Reply:
yea. just abstracted/centralized everything to make this easier to maintain